### PR TITLE
shortcut for getting 4 directions(right,left,up,down) Fixed

### PR DESCRIPTION
**Pull request by Raader**
updated vector2.cs file.
Added 4 public static properties
(Up,Down,Right,Left).
this a fixed version of the old pull request

### CODE REVIEW

File: MonoGame.Framework/Vector2.cs [hunk 1]
```diff
@@ -25,7 +25,10 @@ public struct Vector2 : IEquatable<Vector2>
         private static readonly Vector2 unitVector = new Vector2(1f, 1f);
         private static readonly Vector2 unitXVector = new Vector2(1f, 0f);
         private static readonly Vector2 unitYVector = new Vector2(0f, 1f);
-
+        private static readonly Vector2 rightVector = new Vector2(1f, 0f);
+        private static readonly Vector2 leftVector = new Vector2(-1f, 0f);
+        private static readonly Vector2 upVector = new Vector2(-1f,0f);
+        private static readonly Vector2 downVector = new Vector2(1f, 0f);
```
Comment (Kwyrky) [inline MonoGame.Framework/Vector2.cs:31]: right down and up left are the same vectors?

Comment (SimonDarksideJ) [reply MonoGame.Framework/Vector2.cs:31]: Any update @Raader on resolving the identified issue?

**Review by SimonDarksideJ — CHANGES_REQUESTED** [review]

**Comment by Raader** [comment]
i stopped working on this because the maintainer didn't think it was necessary and it would just be performance loss and it can be made easily with a helper class if the user wants to.

**Comment by SimonDarksideJ** [comment]
OK, can you then close this PR and withdraw it.

**Pull request closed without merging**